src/components/Tag.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import renderer from "react-test-renderer";
```

Comment:
```suggestion
import { create } from "react-test-renderer";
```

In general, let's do this. Don't feel like you need to change this one unless you are in there anyway.